amy/consents/migrations/0009_term_training_request_content.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.2.18 on 2023-04-05 10:56
+
+from django.db import migrations, models
+
+term_training_request_content = {
+    "may-contact": (
+        "The Carpentries will contact you about the progress of your application."
+        " Once certified, may we also contact you for volunteering opportunities,"
+        " surveys, feedback, etc.?"
```

Comment:
@maneesha @froggleston seeking your feedback about this phrasing. This will be displayed for the may-contact consent on the instructor training request form (unchanged elsewhere)

Reply:
The only change I might make would be `other volunteering opportunities`